src/permission/types.py

```diff
@@ -29,6 +31,21 @@ async def has_permission(db_session: database.DBSession, computing_id: str) -> b
 
         return False
 
+class ElectionOfficer:
+    @staticmethod
+    async def has_permission(db_session: database.DBSession, computing_id: str) -> bool:
+        """
+        An current elections officer has access to all elections, prior elections officers have no access.
+        """
+        officer_terms = await officers.crud.current_executive_team(db_session, True)
+        current_election_officer = officer_terms.get(officers.constants.OfficerPosition.ElectionsOfficer.value)[0]
```

Comment:
what if there are multiple active elections officers? I can imagine this scenario. Instead of using the function `current_officers`, instead get the list of all active officer positions a user has & check if one of them is elections officer. This can be done more easily using `get_officer_terms`